content/_layouts/base.njk

```diff
@@ -43,6 +43,9 @@
 
     {% include 'site/feed.njk' %}
 
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="#d9e4e7">
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#632645">
```

Comment:
I'm curious how these colors were chosen (esp for dark mode) -- that is, do they match other existing colors in our palette, or more "it looks good"?

Reply:
Here you can see the two colors I chose. 
The dark raspberry color shows up on hover. 
The light blue color is the background on quotes.

<img width="1440" alt="Screen Shot 2021-06-25 at 12 06 45 PM" src="https://user-images.githubusercontent.com/8933115/123467981-65919e80-d5ae-11eb-8183-d2c2f1899b4c.png">

Reply:
the light/dark blue may be more clear for our brand, since that's what we use for the logo?